Modules/Core/Common/include/itkConceptChecking.h

```diff
@@ -604,7 +604,7 @@ struct IncrementDecrementOperators
     constraints()
     {
       ++a;
-      a--;
+      --a;
```

Comment:
It is funny this wasn't done in the first pass :D

Reply:
Hmmm... I think I should not have modified `IncrementDecrementOperators::constraints()` anyway. I guess it's part of the constraints that both `a--` and `--a` are supported!

Reply:
Update: with the [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/2021f7ce8395e8471df58237408a48fcc167e890..917355e41da7b4969d73f2b5b078675651c87c90) that I just did, I undid those itkConceptChecking.h changes, and squashed the two commits.